Havtorn/Source/Editor/Windows/InspectorWindow.cpp

```diff
@@ -131,6 +133,9 @@ namespace ImGui
 		ImGui::DragFloat3("Rotation", matrixRotation, SlideSpeed);
 		ImGui::DragFloat3("Scale", matrixScale, SlideSpeed);
 		ImGuizmo::RecomposeMatrixFromComponents(matrixTranslation, matrixRotation, matrixScale, transformMatrix.data);
+
+		// TODO.NR: Fix rotation here, can't rotate full range of Yaw due to weird clamping. Other axes are fine
```

Comment:
Found this Git-Issue from ImGuizmo's repo regarding this specific issue. Or I believe it's the same or similar issue!
https://github.com/CedricGuillemet/ImGuizmo/issues/244